config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-ci.yaml

```diff
@@ -304,6 +336,36 @@ postsubmits:
             privileged: true
     annotations:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-aws
-      testgrid-tab-name: bazel-ci-e2e-release-0-4
+      testgrid-tab-name: ci-e2e-release-0-4
+      testgrid-num-columns-recent: '20'
+      testgrid-alert-email: kubernetes-sig-cluster-lifecycle-cluster-api-alerts@googlegroups.com
+  - name: ci-cluster-api-provider-aws-e2e-conformance-release-0-4
+    path_alias: "sigs.k8s.io/cluster-api-provider-aws"
+    decorate: true
+    max_concurrency: 5
+    labels:
+      preset-dind-enabled: "true"
+      preset-kind-volume-mounts: "true"
+      preset-service-account: "true"
+      preset-aws-ssh: "true"
+      preset-aws-credential: "true"
+    branches:
+      - release-0.4
+    spec:
+      containers:
+        - image: gcr.io/k8s-testimages/kubekins-e2e:v20191112-9f04410-1.16
+          command:
+            - "runner.sh"
+            - "./scripts/ci-e2e-conformance.sh"
```

Comment:
Should this be ci-conformance.sh?

Reply:
No, see https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/1360/files#diff-a2c024158251d83fd27abc4413253616R1